src/app/work-item/work-item.service.ts

```diff
@@ -198,8 +199,11 @@ export class WorkItemService {
     if (this.prevFilters.length) {
       for (let i = 0; i < this.prevFilters.length; i++) {
         // In case of assignee filter
-        if (this.prevFilters[i].id === 1 && this.prevFilters[i].active === true) {
-          if (workItem.relationalData.assignees.findIndex(item => item.id == this.prevFilters[i].value) === -1) {
+        if (this.prevFilters[i].id === 1
+            && this.prevFilters[i].active === true) {
+          if (typeof(workItem.relationships.assignees.data) === 'undefined' // If un-assigned
+              || workItem.relationships.assignees.data.findIndex(item => item.id == this.prevFilters[i].value) === -1 // If assignee is not current
```

Comment:
Try to evaluate the LHS before entering the conditional block, preferably just do the comparison within `( ... )`

I mean to say,

```
let assignees = workItem.relationships.assignees;

if (!assignees.data || assignees.data.findIndex(...) === -1) {
  ...
}
```

I think `!workItem.relationships.assignees.data` is vastly superior than `typeof(workItem.relationships.assignees.data) === 'undefined'` conditional and I also think the arrow function may not be apt/needed here, but I'm not 100% confident what are you trying to do with `item.id == this.prevFilters[i].value` comparison.